firmware/components/uart_imu/uart_imu.c

```diff
@@ -214,6 +229,7 @@ int imu_init()
   /*
     75 65 01 02 02 02 E1 C7                           // Put the Device in Idle Mode
     75 65 0C 0A 0A 08 01 02 04 00 01 05 00 01 10 73   // IMU data: acc+gyr at 1000Hz
+    75 65 0C 0A 0A 0A 01 02 05 00 01 0D 00 01 1B A3   // EF data: RPY + LinACC at 500Hz (max)
```

Comment:
Out of curiosity, how did you come up with this numbers?

Reply:
This is hand made from the datasheet here:  https://s3.amazonaws.com/download.microstrain.com/Manuals/3DM-CX5-25+DCP+Manual+(8500-0085).pdf

I then computed the CRC with a simple C program.

Ideally, this would constructed by the driver, but I don't want to spend too much time for something that probably wont change..

Reply:
Thanks for the link - that's totally fine to have it the way it is for now!